vtr_flow/tasks/regression_tests/vtr_reg_basic/hdl_include_yosys/config/config.txt

```diff
@@ -0,0 +1,41 @@
+#################################################################
+# Configuration file for running experiments                    #
+#                                                               #
+# This config file is testing the ability to specify include    #
+# files that should pass to the VTR frontend with the top       #
+# module of the benchmark (ch_intrinsic_top.v). This is done    #
```

Comment:
Same typo comments and questions as in the review comments above.

Also, it seems this comment block should mention this test is to check include files with the yosys front end?